### PR TITLE
Check 101 - issue  #1054

### PR DESCRIPTION
**Pull request by AlexandreHT**
Prefer IS NOT to NOT IS - IF NOT methodcall(). - false positive

### CODE REVIEW

File: src/checks/zcl_aoc_check_101.clas.testclasses.abap [hunk 1]
```diff
@@ -202,4 +206,54 @@ CLASS ltcl_test IMPLEMENTATION.
...
+* ===========
+
+    _code 'if not lcl_program->some_check( iv_param = lv_param ).'.
+    _code '  "some code'.
+    _code 'endif.'.
+
+    ms_result = zcl_aoc_unit_test=>check( mt_code ).
+
+    cl_abap_unit_assert=>assert_initial( ms_result ).
+  ENDMETHOD.
```
Comment (pcf0) [inline src/checks/zcl_aoc_check_101.clas.testclasses.abap:257]: To get the complete picture, could you also add two checks without parameters?

```abap
    _code 'if not lcl_program=>some_check( ).'.
    _code '  "some code'.
    _code 'endif.'.
```

```abap
    _code 'if not lcl_program->some_check( ).'.
    _code '  "some code'.
    _code 'endif.'.
```

**Comment by AlexandreHT** [comment]
Good idea. Done.

**Comment by larshp** [comment]
#1054

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**